ibis/backends/risingwave/__init__.py

```diff
@@ -268,3 +316,261 @@ def list_databases(
             databases = list(map(itemgetter(0), cur))
 
         return self._filter_with_like(databases, like)
+
+    def create_materialized_view(
```

Comment:
Can you add `@experimental` here? This will indicate that the method may change between non major releases.

Reply:
added.